src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -203,6 +241,12 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
      * @return the final set of type to validate. May be empty.
      */
     private Collection<ValidationType> calculateValidationTypesToApply(final List<ValidationType> excludeTypes) {
+        if (VALIDATE_GVCF && !excludeTypes.contains(ValidationType.ALLELES)) {
```

Comment:
I think if we added GVCF to the validation enum then wouldn't everyone need to exclude it in their command lines when validating vcfs?